integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -154,28 +153,121 @@ The REST API Gateway ensures all requests are authenticated with SigV4. Any Infl
 
 ## IAM Permissions
 
-### IAM Deployment Permissions
+The following permissions are the least required permissions for deploying and executing the connector. These permissions assume the stack is named `InfluxDBTimestreamConnector`.
```

Comment:
What does the user need to do with these permissions?

Reply:
The user needs to have these permissions, allowing them to deploy the connector with the SAM CLI and later send requests to the REST API Gateway endpoint. Should I make it clearer that users need these permissions?

Reply:
Yes, it's not clear to me what to do with this. Am I expected to setup an IAM user with these permissions, and have an aws-cli profile that uses it?

Reply:
Yes, this is the expectation. The CloudFormation template can only do as much as the user deploying it has permission to do. A user needs these permissions **at minimum**. They don't need to necessarily create a new policy.

Reply:
I think a CF template to set up these role would be worthwhile.